build/assets/file_type_solidity.svg

```diff
@@ -0,0 +1,8 @@
HC7r6lY8vxTq/sIzqaWufTKK1dbHcA5wY6+Sxn7RdgnZxiiHHrrlQbkFP/WO4wMz+K9bJowadOuuiWUn8H3KxTKL3YLt2JxGv5iCBL3Yz+x/gsthz9EjOXXOh3IWqw7r0GTOL+oZK9KCrfRGtXu/KatLbiLpr6qzrzo2Axx4iiaCr0/7i2HP0wO71f6S6T7cx9hA9cKqZRf/3nETQ1b5M9V6itKOUrrlQtIi4fT8zs0/HHgISCjozux0Ivp+j3GsJ1f05uc88qvcuJbyttLjZzaCSCbraN9DenG2YoF+Lg5saR19vbfmWmSVzOyqpoDOzzVSb6UhYyymvay6USfTGRGh302HXXBNJBR2AmV0K/Dz2HAWZpHoLQmZ2O1pEHNJZZpbU93ByQVc7A33hhVJy11wok2i5SSi30XHXXBNJfgOY2dXAD2LPUYA+dM2FsoXqVy4ZzcfqdqKkJBl0tbPQF96o+tA1F8oEuqob1a+BD8QeYjrJBp2ZrQMujD1HxvrUNRfKONp/eBRRuuaaSDboahegdU7D6FvXXCjjaNewYf2XmZ0be4iZJB109asjUYr6MrcX/euaC2U+cHPsITITtWuuiaSDrvYt9KrOIPrcNRfCGAq6QX3VzH4Ye4jZJB90ZnYPVbuJNLOS6sa6DE+ddc1tBN4Ve4i5JB90AGZ2OXB57DkyoK65cDaizrom/tHMkv+NK4ugq6mzbm6rUNdcKOqsm9tNJNA110Q2QVf/1PhW7DkStjuwOPYQBbor9gAJe5+ZZVGtlk3Q1c5FZYnTUddcO8bRg7CZ/NzMPhV7iKayCjozu41qbZ1sT11z7VFn3c62AifFHmIQWQVd7UKqewNS0eLgdo2hzrod/YeZ/VvsIQaRXdCZ2d3A2bHnSMgK1DXXtknghthDJGIz8M7YQwwqu6Cr/QD4RewhEqCuue6os65yjpn9NPYQg8oy6LTx9b3UNded+ejBxG1kdm9um2y/SeoG00tjzxGRuua6dzfVr2599YkUu+aayDboamfR3y88LSfpXp8XEV8HvD/2EMPKOujM7EaqncP6Zg9gUewhemqMfu4//H/rzauylHXQ1c6nulHcF1ocHFcfO+t+bGZnxh5iFNkHnZndSb8665ahrrnY5tGfKqd7SGzrwmFkH3S1i+lHrY665tLQp866C+stSLNWRNDVnXVZX1o3tAJ1zaVikvJ/hb2LxJuDmyoi6ADM7DLgithztGgBVUW6pGMDZVeH/ZOZFfGUuZigq32Rclevr0Zdc6mZT7nLTdYBfxV7iFCKCjoz+zVwSew5WrAb6ppL1T3ApthDtOADuXTNNVFU0NW+RHlliVpOkq4Jyns17Bdm9onYQ4RUXNCZ2a3AV2LPEdBSYGHsIWRWE1TvgZYiy/dZZ1Nc0NW+RhmP/8epdvWStI1RTo3Tf5jZ12MPEVqRQWdmm4BzYs8RwHKqG96SvknyL+jMsmuuiSKDrvZd4JexhxjBfKqgk3ysJ++n/uea2ZWxh2hDsUFXQGeduubyM0nV8pGj24H3xB6iLUV/I5nZz4AfxZ5jCAupNryR/Gwiz42vP2lmxe7FUnTQ1c4kvy+8NbEHkKHNA34de4gB/RY4LfYQbSo+6MxsLXBR7DkGsBh1zeVuDLgz9hAD8HrTqWIVH3S1fyGPskR1zZVhnHzadC43szNiD9G2XgSdmd0BnBd7jgb2onp5X/I3D7g19hBz2Aq8K/YQXehF0NUuAtbGHmIWE2hxcEnGSH+j9W+Y2XdjD9GF3gSdmW0h7c66lahrrjSTpPvD9S4KaA5uqjdBB2BmPwJS3HxXXXPluoM0O+s+b2a/ij1EV3oVdLUvkN7qdXXNlSvFzrqbgb+IPUSXehd0ZnYN1ethqVDXXPlS66w7vX5A1xu9C7raOaTzhaflJOVLqbPuauDjsYfoWi+Dzsxupqpyim0J6prri3Gq+3WxnVxvJtUrvQy62leIu85pnOrFfemHFDa+/p6ZfTXyDFH0NujM7C6q2vVY1DXXP5NUm87EsIUeLSfZUW+DrnYJcV7AVtdcf91CnKf+55nZTyKcNwm9DrqInXUr6fm/+x6bpPv3YO+gwH0gBtH7bzYz+x/gsg5PuZBqwxvpr7votjrsU2ZWyp4WQ+l90NXOpLvV61pOIvPobhHx9cApHZ0rWQo6wMx+C1zcwakWUy0QFgHY2ME5/qb0rrkmFHT3+TLtdtapa06mGqf9/SWuMLPPtXyOLCjoamZ2O/CvLZ5CXXOyowmqncPa0JuuuSYUdNv7Bu3szTkBrGjhuJK3cdrbC/YiM/tOS8fOjoJuCjPbDJzVwqFXUN2AFtlRG511m+jx4uDpKOh2YGaXAj8PeMgFwLKAx5Py3EHYRcRnmFnOm7cHp6CbXsjOulWoa05mN59w7Sa30LOuuSYUdNMws18APwhwqEXAHgGOI+XbDIRYBvIhM7stwHGKoqCb2VmM/oWn5STS1ASjLyL+JfDRALMUR0E3AzNbB1w4wiGWALsGGkf6YdTOuj/vY9dcEwq62V0ADPNrgLrmZBjjDP8E9gdm1uY60Kwp6GZhZhuBc4f41GWoa06GM49q85pBbAFObGGWYijo5vYtBnsiNg8tDpbhjTF40J1vZj9uY5hSKOjmUN/z+OIAn7IK/XuV0QzSWbcBeHd7o5RB35ANmNnlwOUNPnQX1DUnYWykWWfdP5hZ6DcriqOga+6LzN1Zt6aLQaQX5jH3LZO1wN92MEv2FHQNmdm1VPfrZqKuOQntHqo24pmcUm/yJHNQ0A3mXKYvS1TXnLRhto2vrzSzz3Y5TM4UdAOoX625YJq/Woq65qQd89h5Lae65gakoBvchcBNU/55gmpXL5E2jAE7bmzzTTP7doxhcqWgG1Ddv3/OlD9S15y0bZL7wk5dc0NQ0A3n+8AvqL4A94o8i/TD7VS/sp5pZlfFHiY3CrohTNn4WouDpSvzgSuB98YeJEf/H6yBGzIFxyolAAAAAElFTkSuQmCC"/>
```

Comment:
You can't use images here, it needs to be a proper svg path so that we can apply the colors to it.

Reply:
I managed to export .svg with paths ...

so if I understand correctly pngs and .svgs are like an alternative to each other?

I'll try removing pngs now and only get it to work with .svg

but you can clarify how this works if you have time, thank you

Reply:
Please refer to the “contributing” section of the readme:
https://github.com/braver/FileIcons#contributing

The png files are generated from the svg’s so that various different color schemes can be applied (e.g. color and greyscale for the “mono” variant of this package).

Reply:
I see now, thank you... will try `cargo run` later today..

one additional question: for `cairo` language I wasn't able to get a vectorized image (for now).

in case this one is missing, can the pngs be included manually which then wouldn't be as customizable?

I'll still look into how to get a proper vector svg for cairo though.

Reply:
> in case this one is missing, can the pngs be included manually which then wouldn't be as customizable?

It's possible to override/extend this package and any kind of icons you want. Or simply check out a clone or fork of it (it's not like we have that many updates you need to keep track of anyway). So sure, if you really need a special icon, or for instance if you want multi-colored icons like the blue-yellow python logo, you can do that. We just don't ship such icons as part of this package.